kubernetes/client/rest.py

```diff
@@ -19,13 +19,21 @@
 import ssl
 
 import certifi
+
```

Comment:
This file is generated using openapi-generator (see the top of this file). Could you send the fix to the upstream?

Reply:
ok